README.md

```diff
@@ -373,3 +386,7 @@ NOTE: You must be a member of the `@substrate` NPM org and must belong to the `D
 #### Publish Calc Package
 
 1. `cd` into `calc/pkg` and `npm login`, then `npm publish`.
+
+## Hardware requirements
+
+Sidecar is a stateless program and thus has no specific hardware requirements to run (disk space, memory, etc.).
```

Comment:
FWIW Sidecar shouldn't really use any disk space. Memory wise, I think the node.js processes default to having an upper bound heap of a little less than 2gb, so 4gb of memory should be sufficient. However many users might run sidecar next to a substrate node in a single machine -- in this case they need to significantly improved specs. We have official validator specs, but I am unsure about archive node specs. If we have recommended archive specs we should link to that. The main variable for a nodes hardware will be disk space, which depends on the chain.

Reply:
It might be useful for me to aggregate the benchmarks we have currently, and post some of the results in conjunction with the hardware specs in order to give people an idea of how sidecar runs. So far all of that is internal, as I don't update the benchmarks manually anymore.

Reply:
@Imod7 I think what you referenced above is perfect, and given the info Zeke provided. During my benchmarks of sidecar using 4 threads over 12 open http connections, I found that sidecar would use a max of 1.1GB of RSS mem. The benchmarks were overloading the cache with every runtime possible on polkadot, so this would be a max use case. The hardware in which i tested the benchmarks on were: 

```
Machine type:
n2-standard-4 (4 vCPUs, 16 GB memory)

CPU Platform:
Intel Cascade Lake

Hard-Disk:
500GB
```